run_tests

```diff
@@ -13,4 +13,5 @@ else
 fi
 
 black --diff
-python3 -m unittest -v "$@"
+coverage run --branch --source=lib,src -m unittest -v "$@"
```

Comment:
```suggestion
coverage run --branch --source=src,lib/charms/prometheus_k8s -m unittest -v "$@"
```
Should coverage exclude external libs?

Reply:
Sounds like a good idea. Particularly so, since unit tests for the charm library already exist :-) .

Reply:
I think Leon's suggestion makes sense though. `lib/charms/prometheus_k8s` ensures you're only running tests for your own libraries, not for others you may import in the future. I'll commit that.